### ISSUE TITLE
Sum(3-1/sqrt(x), (x,0,oo)).doit() is still sum

### ISSUE THREAD

**Issue by proy87** [sympy/sympy#14410]
`Sum(3-1/sqrt(x), (x,0,oo)).doit()` gives sum instead of `oo`

**Comment by normalhuman** [comment]
It would be better to start summation with 1 here. That said, `Sum(1/sqrt(x), (x, 1, oo)).doit()` is not evaluated either.

**Comment by jksuom** [comment]
However,
```
>>> Sum(1/sqrt(x), (x, 1, oo)).is_convergent()
False
```

**Comment by normalhuman** [comment]
Yes, but there are so many ways a series can fail to converge; getting `oo` or `-oo` from `doit` gives more information. For example,
```
>>> Sum(1/x, (x, 1, oo)).doit()
oo
>>> Sum(3 - 1/x, (x, 1, oo)).doit()
oo
```
Alas, the second example isn't right...

**Comment by proy87** [comment]
Why `Sum(3 - 1/x, (x, 1, oo)).doit()=oo` is incorrect?

**Comment by normalhuman** [comment]
It should be  `-oo`. SymPy has positive infinity `oo`, negative infinity `-oo` and complex infinity (with no particular direction) `zoo`.

**Comment by proy87** [comment]
`3-1/x > 0` for `x>0`, so how sum of positive terms can be negative infinity?

**Comment by normalhuman** [comment]
You are right. (I didn't have my morning coffee yet.)
